doc/dependencies.md

```diff
@@ -12,6 +12,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | fontconfig | [2.12.1](https://www.freedesktop.org/software/fontconfig/release/) |  | No | Yes |  |
 | FreeType | [2.7.1](https://download.savannah.gnu.org/releases/freetype) |  | No |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk) (Android only) |
 | GCC |  | [7+](https://gcc.gnu.org/) (C++17 support) |  |  |  |
+| glibc | | 2.17 |  |  |  |  |
```

Comment:
you could add a link to https://www.gnu.org/software/libc/

Reply:
I've added a link.